bokehjs/src/coffee/ar/transforms.coffee

```diff
@@ -0,0 +1,113 @@
+HasProperties = require "../common/has_properties"
+
+class Transform extends HasProperties
+
+class AutoEncode extends Transform
+  type: "AutoEncode"
+
+class BinarySegment extends Transform
+  type: "BinarySegment"
+
+class Const extends Transform
+  type: "Const"
+
+class Contour extends Transform
+  type: "Contour"
+
+class Count extends Transform
+  type: "Count"
+
+class CountCategories extends Transform
+  type: "CountCategories"
+
+class Cuberoot extends Transform
+  type: "Cuberoot"
+
+class Encode extends Transform
+  type: "Encode"
+
+class HDAlpha extends Transform
+  type: "HDAlpha"
+
+class Id extends Transform
+  type: "Id"
+
+class Interpolate extends Transform
+  type: "Interpolate"
+
+class InterpolateColor extends Transform
+  type: "InterpolateColor"
+
+class Log extends Transform
+  type: "Log"
+
+class NonZero extends Transform
+  type: "NonZero"
+
+class Ratio extends Transform
+  type: "Ratio"
+
+class Seq extends Transform
+  type: "Seq"
+
+class Spread extends Transform
+  type: "Spread"
+
+class ToCounts extends Transform
+  type: "ToCounts"
+
+module.exports =
+
+  AutoEncode:
+    Model: AutoEncode
```

Comment:
Is there a reason for the `Model` here instead of using `AutoEncode`?  Having all the exposed values under one name require either this nesting or one model per file.